inc/admin/ui/enqueue.php

```diff
@@ -11,6 +11,11 @@ function rocket_add_admin_css_js() {
 	wp_enqueue_style( 'wpr-admin', WP_ROCKET_ASSETS_CSS_URL . 'wpr-admin.css', null, WP_ROCKET_VERSION );
 	wp_enqueue_script( 'micromodal', WP_ROCKET_ASSETS_JS_URL . 'micromodal.min.js', null, '0.4.2', true );
 	wp_enqueue_script( 'wpr-admin', WP_ROCKET_ASSETS_JS_URL . 'wpr-admin.js', [ 'micromodal' ], WP_ROCKET_VERSION, true );
+
+	wp_enqueue_script( 'react' );
+	wp_enqueue_script( 'react-dom' );
+	wp_enqueue_script( 'wp-api-fetch' );
```

Comment:
shouldn't each of those be added as dependencies to the enqueue script in the subscriber instead? Right now I see there is only a dependency on `react-dom`

Reply:
@Tabrisrp I was thinking about this one, however if we plan to add other  React components to WP Rocket, then we need to move this globally.
So, even if now only RUCSS uses React, in the future we might want to use React and React dependencies can be considered as `global` WP Rocket `dependencies`.